examples/quantization/ryzenai/README.md

```diff
@@ -0,0 +1,38 @@
+# Quantization
+Optimum-amd provides a tool that enables you to apply quantization on many models hosted on the Hugging Face Hub using our RyzenAIOnnxQuantizer.
+
+## Static quantization
+The quantization process is abstracted via the AutoQuantizationConfig and the RyzenAIOnnxQuantizer classes. The former allows you to specify how quantization should be done, while the latter effectively handles quantization.
+
+You can read the [VAI_Q_ONNX user guide](https://gitenterprise.xilinx.com/VitisAI-CP/vai_q_onnx/blob/dev/README.md) to learn about VAI_Q_ONNX quantization.

```

Comment:
Can you add the link to the following guide to quantization: https://huggingface.co/docs/optimum/main/en/amd/ryzenai/usage_guides/quantization or https://ryzenai.docs.amd.com/en/latest/vai_quant/vai_q_onnx.html